src/test/app/Flow_test.cpp

```diff
@@ -1262,35 +1290,50 @@ struct Flow_test : public beast::unit_test::suite
         testRIPD1449(true);
         testRIPD1449(false);
 
-        auto testWithFeats = [this](auto&&... fs)
-        {
-            testLineQuality({fs...});
-            testFalseDry({fs...});
-            if (!sizeof...(fs))
-                return;
-            testDirectStep({fs...});
-            testBookStep({fs...});
-            testDirectStep({featureOwnerPaysFee, fs...});
-            testBookStep({featureOwnerPaysFee, fs...});
-            testTransferRate({featureOwnerPaysFee, fs...});
-            testSelfPayment1({fs...});
-            testSelfPayment2({fs...});
-            testSelfFundedXRPEndpoint(false, {fs...});
-            testSelfFundedXRPEndpoint(true, {fs...});
-            testUnfundedOffer(true, {fs...});
-            testUnfundedOffer(false,  {fs...});
-            testReexecuteDirectStep({fix1368, fs...});
-            testSelfPayLowQualityOffer({fs...});
-        };
+        testWithFeats(false, featureFeeEscalation, fix1513);
+        testWithFeats(featureFlow, featureFeeEscalation, fix1513);
+        testWithFeats(featureFlow, fix1373, featureFeeEscalation, fix1513);
+        testWithFeats(featureFlow, fix1373, featureFlowCross,
+            featureFeeEscalation, fix1513);
+        testEmptyStrand({featureFlow, fix1373, featureFlowCross,
+            featureFeeEscalation, fix1513 });
+    }
+};
+
+struct Flow_manual_test : public Flow_test
+{
+    void run() override
+    {
```

Comment:
I like this change a lot. Good thinking!

Reply:
Thanks!